### PR TITLE
Publish Monthly RubyGems Update for February 2023

### PR DESCRIPTION
**Pull request by lauragift21**
RubyGems update roundup for February 2023.

Hi @deivid-rodriguez, can you help review this month's update to ensure I accurately captured the work done last month?

### CODE REVIEW

**Review by deivid-rodriguez — COMMENTED** [review]
RubyGems update looks good. Some improvements, even if worked on during February, did not make it into a February release, so the wording is not totally accurate. These are PRs #6339, #6381 (this is just a small fix in our release scripts, maybe not worth including at all), #6327, and #6384.

Perpahs we could move those to a separate paragraphs and introduce them as:

> Other improvements we worked on during this month, but that did not make it into a February release are:

Also, I think it's definitely worth mentioning @segiddins' work at https://github.com/rubygems/rubygems/pull/6309 too!

**Comment by simi** [comment]
What to include also some rubygems.org infrastructural updates? It could be worth to mention update of terraform (0.13.7 -> 1.3.9), update of AWS terraform (2.51 -> 4.54), update of other providers (external 1.2 -> 2.2, kubernetes 1.8 -> 2.18, template 2.1 -> 2.3) has happened.  Datadog APM was integrated. All RDS instances were migrated to terraform and testing of managed node groups migration has happened in February as well.

**Comment by lauragift21** [comment]
Thanks, @simi and @deivid-rodriguez! I'll include these in the update too.

File: _posts/2023-03-08-february-rubygems-updates.md [hunk 1]
```diff
@@ -19,28 +19,32 @@ The following improvements and fixes are included in these releases (see the cha
...
 
-- set up datadog for application performance monitoring
 - added telemetry to capture MFA login durations - [#3376](https://github.com/rubygems/rubygems.org/pull/3376).
+- integrated DataDog for application performance monitoring - [#3461](https://github.com/rubygems/rubygems.org/pull/3461)
 - set up GitHub OAuth to protect the new /admin namespace - [#3388](https://github.com/rubygems/rubygems.org/pull/3388).
 - updated Rails test job name to be stable across version updates - [#3420](https://github.com/rubygems/rubygems.org/pull/3420).
 - fixed test avo warnings by removing redundant rake tasks loading - [#3422](https://github.com/rubygems/rubygems.org/pull/3422).
 - added avo MFA reset admin action & view of audit entries - [#3426](https://github.com/rubygems/rubygems.org/pull/3426).
 - updated `erd.dot` to fix ERD CI - [#3490](https://github.com/rubygems/rubygems.org/pull/3490).
+- updated the following packages - terraform `0.13.7 -> 1.3.9`, AWS terraform `2.51 -> 4.54`, external `1.2 -> 2.2`, kubernetes `1.8 -> 2.18`, and template `2.1 -> 2.3`.
```
Comment (simi) [inline _posts/2023-03-08-february-rubygems-updates.md:47]: There was terraform upgraded and also individual terraform providers. I think it is ok to make this into 2 individual points.

```
- updated terraform `0.13.7 -> 1.3.9`
- updated terraform providers
  - AWS `2.51 -> 4.54`
  - external `1.2 -> 2.2`
  - kubernetes `1.8 -> 2.18`
  - template `2.1 -> 2.3`.
```

File: _posts/2023-03-08-february-rubygems-updates.md [hunk 2]
```diff
@@ -19,28 +19,32 @@ The following improvements and fixes are included in these releases (see the cha
...
 
 In February, RubyGems gained [114 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2023-02-01%7D...master@%7B2023-02-28%7D), contributed by 16 authors. There were 1,767 additions and 234 deletions across 106 files.
 
 ## RubyGems.org News
 
-This month, we made a lot of progress working on the backend admin dashboard. We implemented robust auditing of all changes, and added support for resetting users MFA, blocking a user, and deleting webhooks. 
+This month, we made significant progress on the backend admin dashboard. We implemented robust auditing of all changes and added support for resetting user's MFA, blocking a user, and deleting webhooks. 
 
-The dependency API deprecation has been [announced](https://blog.rubygems.org/2023/02/22/dependency-api-deprecation.html), and we have a plan for brownouts & turning off the endpoint entirely.
+We have [announced](https://blog.rubygems.org/2023/02/22/dependency-api-deprecation.html) the deprecation of the dependency API, and we plan to implement brownouts and remove the endpoint entirely. We migrated all RDS instances to Terraform and tested the migration of managed node groups.
```
Comment (simi) [inline _posts/2023-03-08-february-rubygems-updates.md:36]: We migrated all RDS instances to be managed by Terraform and tested the migration of managed node groups on rubygems.org EKS cluster.

**Review by simi — APPROVED** [review]

**Pull request merged**